.comate/specs/quran-uthmani-viewer/doc.md

```
@@ -0,0 +1,176 @@
+- `itm-ikhlas-*`
+- `itm-falaq-*`
+- `itm-nas-*`
+- `itm-ayat-al-kursi-*`
+- `itm-recitation-*`
+- `itm-ayatain-*`
+
+This approach requires no schema migration and covers all current and future Quran items following the established naming convention.
+
+### Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)